packages/@aws-cdk/aws-ivs-alpha/lib/channel.ts

```diff
@@ -53,22 +53,53 @@ export enum LatencyMode {
 /**
   * The channel type, which determines the allowable resolution and bitrate.
   * If you exceed the allowable resolution or bitrate, the stream probably will disconnect immediately.
+  *
+  * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
 */
 export enum ChannelType {
   /**
-   * Multiple qualities are generated from the original input, to automatically give viewers the best experience for
-   * their devices and network conditions.
-   *
-   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   * Multiple qualities are generated from the original input, to automatically give viewers the best experience for their devices and network conditions.
+   * Transcoding allows higher playback quality across a range of download speeds. Resolution can be up to 1080p and bitrate can be up to 8.5 Mbps.
+   * Audio is transcoded only for renditions 360p and below; above that, audio is passed through.
    */
   STANDARD = 'STANDARD',
 
   /**
    * delivers the original input to viewers. The viewer’s video-quality choice is limited to the original input.
```

Comment:
Not a change you made, but while we're here:

```suggestion
   * Delivers the original input to viewers. The viewer’s video-quality choice is limited to the original input.
```